src/javascript/binary/base/client.js

```diff
@@ -324,7 +324,7 @@ const Client = (() => {
     const getUpgradeInfo = (landing_company, jp_account_status = State.getResponse('get_settings.jp_account_status.status')) => {
         let type         = 'real';
         let can_upgrade  = false;
-        let upgrade_link = 'realws';
+        let upgrade_link = 'welcome';
```

Comment:
This function is being called from many places to get the upgrade link, etc. (i.e. the upgrade link in header needs this).
I'd suggest to change it [here](https://github.com/binary-com/binary-static/blob/master/src/javascript/binary/websocket_pages/user/new_account/virtual_acc_opening.js#L115) instead (else part).

Reply:
Alright, previously it was changed in `virtual_account_opening.js`. I'll just revert back. Thanks